pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -210,4 +210,5 @@ def shutdown(self):
         """ Ensures the instrument in a safe state """
         self.voltage = 0.
         self.isShutdown = True
+        super().shutdown()

```

Comment:
@LongnoseRob, as you mentioned in the overall comment, I would remove line 212, but also 214. They are both included in `super`.